src/components/AccountDrawer/PrefetchBalancesWrapper.tsx

```diff
@@ -44,7 +45,7 @@ export default function PrefetchBalancesWrapper({ children }: PropsWithChildren)
   const [hasUnfetchedBalances, setHasUnfetchedBalances] = useState(true)
   const fetchBalances = useCallback(() => {
     if (account) {
-      prefetchPortfolioBalances({ variables: { ownerAddress: account } })
+      prefetchPortfolioBalances({ variables: { ownerAddress: account, chains: GQL_MAINNET_CHAINS.concat() } })
```

Comment:
why `concat()` with no second array as an argument? if you're trying to make a shallow copy here i think there's more canonical ways - `.slice()` would be my go to

Reply:
it was intended to be a mutable copy of the Readonly array. Based on the these discussion posts concat and slice seem interchangeable, I'm fine with making our standard `slice`
- https://stackoverflow.com/questions/58981365/converting-a-readonlyarrayany-to-a-usual-mutable-array
- https://stackoverflow.com/questions/17803586/javascript-array-copying-concat-vs-slice-which-one-is-better